### PR TITLE
Desktoppro

### PR DESCRIPTION
**Pull request by KAYEMBAJONATHAN**
- In general I worked on my desktop project with my html for the UI 
- I add style in my css for the project to look clean and better 
- i also add my assets folder to keep my i mages so that it can display on the card ...

### CODE REVIEW

**Comment by KAYEMBAJONATHAN** [comment]
linters fixed

**Review by BregornOriginal — CHANGES_REQUESTED** [review]
## Hi @KAYEMBAJONATHAN  ❌ 

### While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

## You have submitted a project, that All main headers from the [README template](https://github.com/microverseinc/readme-template) should be present

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by KAYEMBAJONATHAN** [comment]
- updated Readme.md
- @BregornOriginal

File: style.css [hunk 1]
```diff
@@ -245,21 +243,18 @@ li {
...
     background-repeat: no-repeat;
     background-size: contain;
     height: 100vh;
     background-position: center;
-    padding-top: 20px;
     background-color: #fff;
-    padding: 24px;
-    
+    padding: 24px 80px;
   }
```
Comment (zieeco) [inline style.css]: - Hey, great job with this section, however, let's align the content on the section well enough to ensure they're close to the Figma design to improve your design. see below:


*your design*
Screenshot from 2022-12-16 18-19-23

*Figma design*
Screenshot from 2022-12-16 18-19-03

Comment (zieeco) [reply style.css]: ### MOBILE

- Please, your mobile design now is been looking different as a result of your desktop implementation. Please, kindly ensure you fix this issue with all the sections in the mobile design to improve the user experience of your portfolio website.

File: style.css [hunk 2]
```diff
@@ -284,7 +278,6 @@ display: none;
     flex-direction: row;
     padding: 2% 8%;
     gap: 5%;
-  
     margin: 24px;
   }
```
Comment (zieeco) [inline style.css]: - [ ]  Hey, your images in each card are not of the same size and this is making your design not consistent. Please, ensure all images are of the same size to improve your design. Kindly check the images on Figma to improve on this issue.

- [ ]  The second and fourth card content are having a reversed direction from the Figma design. You're yet to add this missing feature. Please, kindly check the card's content closely to see their directions and improve your design.
- [ ] Kindly maintain the font size, weight, and colors on each card content to improve your design.
- [ ] Kindly maintain the correct styles on the list items in each card to match the Figma design as much as you can.
- [ ] Kindly add the missing interaction states on each button to indicate when it's on `hover, active, enabled, disabled`. You can check the Figma design to use the right color indicating the different states to improve the user experience of the website.


---

- [ ]  Hey, please, let's adjust our on cards so they have some spaces left and right and not taking the whole width of this project section thereby making it close to the Figma design as much as you can below:

- [ ]  Also, kindly remove the space before each image, as they don't seem to have a space in the container. see the images below to improve your design.


*your design*
Screenshot from 2022-12-16 18-49-19


*Figma design*
Screenshot from 2022-12-16 17-07-35

File: style.css [hunk 3]
```diff
@@ -304,51 +298,73 @@ display: none;
...
+  }
 
-.header-img > h1{
-  color: #fff;
-}
+  #about {
+    display: flex;
+    gap: 20px;
+    color: #344563;
+  }
```
Comment (zieeco) [inline style.css]: - [ ] Kindly maintain the font size, weight, and colors on the `language, skill, framework`.
- [ ] Let's add the missing drop-down image icon at the right side in the same line as the  `Languages`
- [ ] Let's also make some adjustments to our styling to improve the spacing of the sides of the about section to make our design close to the Figma design as much as we can below:



*Figma design*
Screenshot from 2022-12-16 18-43-34

*your design*
Screenshot from 2022-12-16 18-41-44

File: style.css [hunk 4]
```diff
@@ -304,51 +298,73 @@ display: none;
...
+    max-width: 45%;
+    margin: 0 auto;
+  }
 
+  .header-img {
+    text-align: center;
+    border-top-left-radius: 120px;
+    padding-top: 2%;
+    background-position: center;
+  }
```
Comment (zieeco) [inline style.css]: - [ ] Hey, please, let's make some adjustments by adding the correct background image and reducing the width of the text so as for the text not to be too much apart from the inputs. I made a highlight in the images below so you can see how to adjust it. Let's fix this to improve our design as much as we can below:
- [ ] Let's remove the black border on the input to improve our design to match the Figma design as much as we can.
- [ ] Let's center the button to improve our design below:


*Figma design*
Screenshot from 2022-12-16 17-22-04

*your design*
Screenshot from 2022-12-16 19-02-30

File: README.md [hunk 5]
```diff
@@ -1 +1,130 @@
...
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Hello_Microverse <a name="about-project"></a>
+
+
+
+**Hello_Microverse**  all about my first web portf
```
Comment (zieeco) [inline README.md]: - Let's kindly update the project name to our current project to make the readme more professional.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @KAYEMBAJONATHAN ,


Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 6]
```diff
@@ -245,21 +237,29 @@ li {
...
     background-repeat: no-repeat;
     background-size: contain;
     height: 100vh;
     background-position: center;
-    padding-top: 20px;
     background-color: #fff;
-    padding: 24px;
-    
+    border-bottom-left-radius: 100px;
+  }
```
Comment (franclobo) [inline style.css]: - [ ] As the last code reviewer said please 🙏🏽 correct the home section, the title and paragraph should be centered to be similar to the Figma template, to be more detailed:
- Correct the font size.
- The background should be responsive.
- Correct the height of this section.

Template-1-home

- [ ] As the last code reviewer said kindly fix the mobile version

https://user-images.githubusercontent.com/58642949/208250253-b8b0906c-1044-4f9d-9121-1b7e64f9900e.mp4

File: style.css [hunk 7]
```diff
@@ -275,17 +275,25 @@ display: none;
...
     flex-direction: row;
     padding: 2% 8%;
     gap: 5%;
-  
-    margin: 24px;
+    margin: 142px 142px 0 142px;
+    align-self: center;
+    padding-left: 25px;
+    width: 80%;
+  }
```
Comment (franclobo) [inline style.css:292]: - [ ] Kindly complete all the required changes from the last code reviewer to look similar to the Figma template 👇🏽 

> Hey, your images in each card are not of the same size and this is making your design not consistent. Please, ensure all images are of the same size to improve your design. Kindly check the images on Figma to improve on this issue.

>  The second and fourth card content are having a reversed direction from the Figma design. You're yet to add this missing feature. Please, kindly check the card's content closely to see their directions and improve your design.

 

> Kindly maintain the font size, weight, and colors on each card content to improve your design.

>  Kindly maintain the correct styles on the list items in each card to match the Figma design as much as you can.

>  Kindly add the missing interaction states on each button to indicate when it's on hover, active, enabled, disabled. You can check the Figma design to use the right color indicating the different states to improve the user experience of the website.

>  Hey, please, let's adjust our on cards so they have some spaces left and right and not taking the whole width of this project section thereby making it close to the Figma design as much as you can below:

>  Also, kindly remove the space before each image, as they don't seem to have a space in the container. see the images below to improve your design.

| Figma template | Your design |
|---|---|
| image | image |

File: style.css [hunk 8]
```diff
@@ -304,51 +313,79 @@ display: none;
...
+
+  .project-item {
+    padding-left: 30px;
+  }
+
+  .about-h1 {
+    padding-top: 0;
+    margin-top: 0;
+  }
+}
```
Comment (franclobo) [inline style.css]: - [ ] Please 🙏🏽 complete the required changes from the last code reviewer, your design must look like the Figma design:

> Kindly maintain the font size, weight, and colors on the language, skill, framework.

>  Let's add the missing drop-down image icon at the right side in the same line as the Languages

>  Let's also make some adjustments to our styling to improve the spacing of the sides of the about section to make our design close to the Figma design as much as we can below:

| Figma design | Your design |
|---|---|
| image | image |

File: style.css [hunk 9]
```diff
@@ -304,51 +313,79 @@ display: none;
...
-  color: #7f8cff;
-  margin-bottom: 0;
-  text-decoration: none;
-}
+  .header-img {
+    text-align: center;
+    border-top-left-radius: 120px;
+    padding-top: 2%;
+    background-position: center;
+  }
```
Comment (franclobo) [inline style.css]: - [ ] Please 🙏🏽 complete the required changes from the last code reviewer, your design must look like the Figma design:

> Hey, please, let's make some adjustments by adding the correct background image and reducing the width of the text so as for the text not to be too much apart from the inputs. I made a highlight in the images below so you can see how to adjust it. Let's fix this to improve our design as much as we can below:
> Let's remove the black border on the input to improve our design to match the Figma design as much as we can.
> Let's center the button to improve our design below:

| Figma design | Your design |
|---|---|
| image | image |

File: README.md [hunk 10]
```diff
@@ -1 +1,130 @@
...
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Hello_Microverse <a name="about-project"></a>
```
Comment (franclobo) [inline README.md:21]: - [ ] Please 🙏🏽 complete the required changes from the last code reviewer:

> Let's kindly update the project name to our current project to make the readme more professional.

image

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @KAYEMBAJONATHAN, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

- No linter errors ✔️ 
- Correct Github flow ✔️ 
- Descriptive PR ✔️ 


 ## Required Changes ♻️
_Check the comments under the review._

- [ ] Please 🙏🏽 complete the required changes from the last code reviewer, you have limited submissions to approve the project.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 6 above)
Comment (KAYEMBAJONATHAN) [reply style.css]: some of your are unfair
@franclobo

File: style.css (hunk 8 above)
Comment (KAYEMBAJONATHAN) [reply style.css]: no you wrong

Comment (KAYEMBAJONATHAN) [reply style.css]: @franclobo

File: index.html [hunk 11]
```diff
@@ -25,14 +25,14 @@
...
     <p>
       I’m AlexandraGlad to see you!I’m a software developer! 
       I can help you build a product , feature or website Look 
       through some of my work and experience! If you like what you 
       see and have a project you need coded, don’t hestiate to contact me.
     </p>
-      <a href="" class="connect">
+      <a href="#" class="connect">
       Let's Connect
     </a>
```
Comment (MeqdamAlqudah) [inline index.html:37]: - [Optional] Please make sure that your design looks exactly like Figma design, I know it is stressful to just make the 100% design match but it is a really valuable skill that all companies search for please fix the sizing in the headline section I suggest you use (VW, it calculate the size depending on the page width) in all your design (width, height, font-size, margin,....) you can use this [website](https://web-development.space/tools/px-to-vw/) to compare all the sizes in the design with your website 
image

File: index.html [hunk 12]
```diff
@@ -89,13 +89,15 @@ <h1 class="work-title">Tonic</h1>
...
            <li>FACEBOOK</li>
-           <li>FULL STACK DEV</li>
-           <li>2015</li>
+           <ul class="details">
+            <li>FULL STACK DEV</li>
+            <li>2015</li>
+          </ul>
        </ul>
        <p>Experimental content creation feature that allows users to 
         add to an existing story over the course of a day without
```
Comment (MeqdamAlqudah) [inline index.html:103]: - [Optional] Please,it is clear that the card size here is fixed try to use more responsive scales  to make it better 
image

**Review by MeqdamAlqudah — APPROVED** [review]
Hi @KAYEMBAJONATHAN , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**